sw/device/silicon_creator/rom_ext/e2e/verified_boot/boot_test.c

```diff
@@ -0,0 +1,36 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/base/status.h"
```

Comment:
This seems similar to some of the programs under `sw/device/silicon_owner/bare_metal`. Would it make sense to consider moving this under that folder?

Reply:
I'd considered that as well, but I feel like the `silicon_owner/bare_metal` subdir is more of an example, whereas `silicon_creator/rom_ext/e2e/...` are definitely tests.